### DISCUSSION TITLE
The clothes that I've gifted to monika are gone after updating manually, HELP

### DISCUSSION THREAD

**Discussion by EColi07** [Monika-After-Story/MonikaModDev#9351] [QA - tech support + general]
Steam, Macbook. 

Since MAS can't automatically update when you're using steam, I followed a tutorial on how to do it manually. I downloaded the new update file (for NOU) and then copy and pasted everything into the original "game" folder, replacing all files that requested to be replaced. When I opened the game all of the clothes that I gave to her are now gone, but for some reason, the ribbons stayed. 

I assume this is because the "mod_assets" folders of the clothes are being replaced along with all the other files that I've pasted into the "game" file, if so, is there a way to avoid this?

The game also crashed a few times before while I was messing around with the codes, and when I restored the game the same thing happens. Thanks for reading :)

**Comment by Booplicate** [comment]
Don't use steam, she warns you it may cause issues. Make a fresh DDLC install and extract the deluxe version of MAS there (that would restore all official spritepacks you gave). Then you can add submods/custom spritepacks there.
